tests/src/helpers.c

```diff
@@ -341,18 +341,26 @@ int mbedtls_test_ascii2uc(const char c, unsigned char *uc)
     return 0;
 }
 
-void mbedtls_test_fail(const char *test, int line_no, const char *filename)
+void mbedtls_test_fail_internal(const char *test, int line_no, const char *filename)
```

Comment:
(For a follow-up once the urgent fix is merged) This function should be either `static`, or listed with documentation in `helpers.h`.

Reply:
Its now static, it doesn't want to have larger scope as that would be dangerous.